net/default_service.go

```diff
@@ -0,0 +1,42 @@
+package net
+
+import (
+	"context"
+
+	"github.com/dedis/drand/protobuf/dkg"
+	"github.com/dedis/drand/protobuf/drand"
+)
+
+type DefaultService struct {
```

Comment:
Try to write a comment explaining what it is, what it does and how to use it please !